### ISSUE TITLE
Some questions (not active but.., user's account phone num, re-send button)

### ISSUE THREAD

**Issue by emresaracoglu** [nextcloud/twofactor_sms#13]
SMS idea is very good! I love it & used.

1. User's TOTP option is **not** active but application require SMS code from user during login page.
2. Instead of manually adding the phone number to the database, Is it possible to use the phone number on the account of the user?
3. Sometimes the SMS code is not sent, can you add re-send button? _(It will be active after 30 sec or 1 minute)_
4. Need to be used with [twofactor_totp](https://github.com/nextcloud/twofactor_totp)?

Thank you @ChristophWurst

**Comment by ChristophWurst** [comment]
> User's TOTP option is not active but application require SMS code from user during login page.

Please elaborate. What is the issue?

> Instead of manually adding the phone number to the database, Is it possible to use the phone number on the account of the user?

Sure that would be possible, that actually makes a lot of sense. No idea how to access that though :-P

> Sometimes the SMS code is not sent, can you add re-send button? (It will be active after 30 sec or 1 minute)

You're right. This app does not really care about error states. That should be fixable.

> Need to be used with twofactor_totp?

Nope, these two apps are totally independent. You can even use both at a time :-)


If you like and use the app I'd really appreciate if you could help with developing and maintaining it. I'm not using it in production, so I'm not spending to much time on this at the moment. Also beware that SMS 2FA is not considered secure anymore.

**Comment by emresaracoglu** [comment]
> `User's TOTP option is not active but application require SMS code from user during login page.`
> Please elaborate. What is the issue?

At first, I only used TWOSF and users who wanted to use it. I later discovered the SMS application and at the same time I started to use both applications. The apps worked fine!

Then I realized, the TOTP code request from users who have not activated the TOTP option... The user does not use the TOTP feature but application want code for login from user.

Later, I dısabled the TOTP application and I just left the SMS application enabled. I am getting Internal Error error this time..
I eventually disabled both applications.

Thanks
